src/nav/apm.yml

```
@@ -210,7 +212,7 @@ pages:
                 path: /docs/apm/agents/java-agent/instrumentation/monitor-deployments-java-agent
               - title: Instrument webpages via API
                 path: /docs/apm/agents/java-agent/instrumentation/instrument-browser-monitoring-java-agent-api
-              - title: Browser best practices
+              - title: Browser monitoring best practices
```

```suggestion
              - title: Browser monitoring best practices in Java
```
Maybe adding the "in Java" will help clarify the difference between this one and the one in the browser monitoring docs.